webapp/src/main/java/org/opentestsystem/rdw/admin/security/auth/DefaultAuthorizationService.java

```diff
@@ -3,13 +3,13 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
This class is where most of the real changes are. 

Yeah, i could try to pull this whole class into RDW_Common but it started to get messy. Maybe next go-around.